### ISSUE TITLE
Searches for upload.cx generally turn up empty despite having results when searching manually

### ISSUE THREAD

**Issue by staridiot** [Jackett/Jackett#15108]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
HTTP Client: HttpWebClient2
Using FlareSolverr: http://*.*.*.*:8191
Using proxy: Disabled
App config/log directory: /home/jackett/.config/Jackett
ThreadPool MaxThreads: 32767 workerThreads, 1000 completionPortThreads
Running in Docker: No
File /etc/issue: Ubuntu 22.04.3 LTS \n \l
Jackett variant: CoreLinuxAmdx64
OS version: Unix 5.15.0.91 (64bit OS) (64bit process)
Environment version: 6.0.27 (/opt/Jackett/)
Starting Jackett v0.21.1897
```

### Description

I'm using the *Arr suite to manage media, I have upload.cx added as a Torznab indexer. I have searched multiple movies within Radarr I found on upload.cx but Jackett states it has returned 0 results for upload.cx.

Which service/program is causing the issue is beyond me, so I apologize if this is not a Jackett issue. The Github states to make a ticket since the Discord was abandoned, normally I wouldn't raise such a seemingly small issue on Github.

### Logged Error Messages

```text
No errors
```

**Comment by garfield69** [comment]
The only way forward is for you to turn on the Jackett enhanced logging, perform a single search with Sonarr for the upload.cx indexer, then upload the full Jackett log so we can see what Sonarr asked for, and how Jackett responded.
See https://github.com/Jackett/Jackett/wiki/Troubleshooting#how-to-provide-a-jackett-enhanced-log-for-a-ticket

**Comment by staridiot** [comment]
I went ahead and followed those instructions, however I did so for Radarr since upload.cx does not have any TV Shows, only movies. And I think I found the issue, I have the 'Search freeleech only' flag set and upload.cx has like 10 releases that show up with the freeleech tag. There's been a global freeleech running but the search doesn't care.

I apologize for my lack of intelligence. This issue is solved.
